lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -347,20 +347,22 @@ static Value promoteOperand(OpBuilder &builder, Location loc, Value operand,
 
 // promote operands of dot op if the existing combination is not natively
 // supported.
-static void decomposeMixedModeDotOp(ModuleOp mod) {
-  mod.walk([](tt::DotOp dotOp) -> void {
+static void decomposeMixedModeDotOp(ModuleOp mod, int computeCapability) {
+  mod.walk([=](tt::DotOp dotOp) -> void {
     auto D = dotOp.getD();
     OpBuilder builder(dotOp);
     Type AElType = dotOp.getA().getType().getElementType();
     Type promoteType;
     NvidiaMmaEncodingAttr mmaLayout =
         D.getType().getEncoding().dyn_cast<NvidiaMmaEncodingAttr>();
     if (mmaLayout) {
-      bool isNativeHopperFP8 =
+      // TODO(hjchen2): Take the cuda version into consideration since float8
```

Comment:
you don't need that as triton explicitly download ptxas version 12.4

Reply:
that's great. I've removed it, thanks